src/oscar/apps/offer/abstract_models.py

```diff
@@ -16,14 +16,15 @@
 from django.utils.timezone import get_current_timezone, now
 from django.utils.translation import ugettext_lazy as _
 
-from oscar.apps.offer import results, utils
-from oscar.apps.offer.managers import ActiveOfferManager
 from oscar.core.compat import AUTH_USER_MODEL
-from oscar.core.loading import get_class, get_model
+from oscar.core.loading import get_class, get_classes, get_model
 from oscar.models import fields
 from oscar.templatetags.currency_filters import currency
 
-BrowsableRangeManager = get_class('offer.managers', 'BrowsableRangeManager')
+ActiveOfferManager, BrowsableRangeManager \
+    = get_classes('offer.managers', ['ActiveOfferManager', 'BrowsableRangeManager'])
+ZERO_DISCOUNT = get_class('offer.results', 'ZERO_DISCOUNT')
+load_proxy, unit_price = get_classes('offer.utils', ['load_proxy', 'unit_price'])
```

Comment:
I have no problem with allowing the overriding of these functions in principle, but I think we need to be adding `get_functions` to do it?

Reply:
Just the name. It would make it explicit what type of object is going to be returned. We could consider `get_object` and `get_objects` as alternatives for use with functions and classes.

Reply:
if that's only the name - I don't see the reason to rename it simply because it's used everywhere through the application to load classes. Also, previously part of the signals where loaded by this method before I refactored them to load directly.

I agree about the necessity to refactor app loading and switch to Django Appconf, but for now this PR will allow to finalize the original idea to make all major imports dynamic for consistency and maximum customizability.